packages/renderer/src/lib/ui/TreeView.svelte

```diff
@@ -0,0 +1,130 @@
+<style>
+.no-arrow {
+  padding-left: 1rem;
+}
+.arrow {
+  cursor: pointer;
+  display: inline-block;
+}
+.arrowDown {
+  transform: rotate(90deg);
+}
```

Comment:
I think we should remove this style directive

for example instead of adding no-arrow class, just add `pl-4`

instead of adding `arrow`, add `cursor-pointer` and `inline-block` classes

etc